library/core/functions.general.php

```diff
@@ -3571,6 +3571,7 @@ function t($code, $default = false) {
      * @see Gdn::translate()
      */
     function translateContent($code, $default = false) {
```

Comment:
If we're going to deprecate a function, we should probably slap a [<span>@</span>deprecated tag](http://docs.phpdoc.org/references/phpdoc/tags/deprecated.html) in its doc block.

Reply:
Done. Check here:
https://github.com/vanilla/vanilla/pull/7784/files